### PR TITLE
symfony 5 compatibility

### PR DESCRIPTION
**Pull request by dbu**
fix #532

### CODE REVIEW

File: src/EventListener/CacheControlListener.php [hunk 1]
```diff
@@ -16,8 +16,16 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\Kernel;
 use Symfony\Component\HttpKernel\KernelEvents;
 
+if (Kernel::VERSION >= 5) {
```
Comment (jderusse) [inline src/EventListener/CacheControlListener.php]: beware, version is a string, what's about using `kernel::MAJOR_VERSION >= 5` or `kernel::VERSION_ID >= 50000`?

Comment (dbu) [reply src/EventListener/CacheControlListener.php]: ups, yes i wanted to use MAJOR_VERSION here, thanks!

**Comment by dbu** [comment]
hm, does anyone have an idea what is going on with the lowest version build here? why is the class loader not finding classes? i verified that the 3.4.5 file exists and contains the expected class (even bumped from 3.4.4 to be sure its not a cached broken archive or something)

**Comment by jderusse** [comment]
@dbu because `UserPasswordValidator` extends `Symfony\Component\Validator\ConstraintValidator` which does not exists (the component is not installed)

**Comment by dbu** [comment]
uh, that is rather implicit. thanks for pointing it out. that was the problem, had to add some dependencies. weird, that. security-bundle says it depends on those, not sure why they did not install.

do you think its ok to merge like this? i don't want to add them to `require` as thats only true for old versions, and the whole thing is require-dev only anyways...

**Comment by jderusse** [comment]
not in 3.4 branch which is installed with prefered-lowest https://github.com/symfony/symfony/blob/3.4/src/Symfony/Bundle/SecurityBundle/composer.json

**Comment by jderusse** [comment]
not in 3.4 branch which is installed with prefered-lowest https://github.com/symfony/symfony/blob/3.4/src/Symfony/Bundle/SecurityBundle/composer.json

File: .travis.yml [hunk 2]
```diff
@@ -60,7 +62,8 @@ matrix:
 before_install:
   - if [[ $COVERAGE != true ]]; then phpenv config-rm xdebug.ini || true; fi
   - if ! [ -z "$STABILITY" ]; then composer config minimum-stability ${STABILITY}; fi;
-  - if ! [ -v "$DEPENDENCIES" ]; then composer require --no-update ${DEPENDENCIES}; fi;
+  - if ! [ -z "$DEPENDENCIES" ]; then composer require --no-update ${DEPENDENCIES}; fi;
```
Comment (jderusse) [inline .travis.yml]: sounds good to me

**Comment by dbu** [comment]
yeah i was looking at require-dev at https://packagist.org/packages/symfony/security-bundle#v3.4.4 :man_facepalming:

**Pull request merged**